chain/stmgr/forks.go

```diff
@@ -292,6 +296,18 @@ func (us UpgradeSchedule) Validate() error {
 	return nil
 }
 
+func (us UpgradeSchedule) GetNtwkVersion(e abi.ChainEpoch) (network.Version, error) {
```

Comment:
@magik6k I added this so that the mpool could get current network version without having to construct a new stmgr and depend on chainstore.  If you think this redundancy is unacceptable I can try a deeper refactor.

Reply:
Seems fine with me